packages/background/src/utils/http.ts

```diff
@@ -70,12 +69,10 @@ const request = async <T>(
             {}
         );
         url += '?' + new URLSearchParams(safeParams).toString();
-    } else {
-        options.body = JSON.stringify(params);
```

Comment:
body is already present in `RequestInit` type. If you wanna do a post you should

```javascript
const someBody = { something: 'cool'};
httpClient.request('https://some.endpoint/', {method:'POST', body: JSON.stringify(someBody)});
```

Reply:
We can validate that the body is undefined in GET request if you wanna more control on that

Reply:
We don't do POST. However, we do have the body because `RequestOptions` extends `RequestInit` 
![image](https://user-images.githubusercontent.com/98833290/225118017-b6970875-93d5-47e3-92ec-90bc2949d9c4.png)
![image](https://user-images.githubusercontent.com/98833290/225118123-90c379b0-c6a8-40b1-bcf7-e3c6ffda8ed6.png)